src/components/Film/Film.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import ModalWatch from "../ModalWatch/ModalWatch";
+import ModalInfoFilm from "../ModalInfoFilm/ModalInfoFilm";
+import ModalTrailer from "../ModalTrailer/ModalTrailer";
+
+import "./style.scss";
+
+const Film = ({
+  title,
+  overview,
+  imgUrl,
+  rating,
+  genreFilmArray,
+  changeGenre,
+  genre
+}) => {
+  const [showInfoFilm, setShowInfoFilm] = useState(false);
+  const [genreFilm, setGenreFilm] = useState([]);
+  const [showTrailer, setShowTrailer] = useState(false);
+
+  useEffect(() => {
+    if (genreFilmArray && genre.length > 0) {
+      setGenreFilm(changeGenre(genreFilmArray));
+    }
+  }, [genreFilmArray, genre]);
+
+  const handleBtnInfo = () => {
+    setShowInfoFilm(true);
+  }
+
+  const handleCloseInfo = () => {
```

Comment:
let's combine handleBtnInfo and handleCloseInfo to toggleInfoModal

`const toggleInfoModal= () => setShowInfoFilm((showInfoFilm) => !showInfoFilm)`

Reply:
done